test/images/dnsutils/BASEIMAGE

```diff
@@ -0,0 +1,4 @@
+amd64=alpine
```

Comment:
https://hub.docker.com/r/arm32v6/alpine/ and https://hub.docker.com/r/arm64v8/alpine/